x-pack/plugins/fleet/server/services/epm/elasticsearch/transform/install.ts

```diff
@@ -151,15 +382,20 @@ async function handleTransformInstall({
       throw err;
     }
   }
-  await esClient.transform.startTransform(
-    { transform_id: transform.installationName },
-    { ignore: [409] }
-  );
+
+  // start transform by default if not set in yml file
+  // else, respect the setting
+  if (startTransform === undefined || startTransform === true) {
+    await esClient.transform.startTransform(
+      { transform_id: transform.installationName },
+      { ignore: [409] }
+    );
+  }
```

Comment:
Let's add debug logs here for whether or not we started the transform

Reply:
Updated here [`8fa543f` (#140046)](https://github.com/elastic/kibana/pull/140046/commits/8fa543f9bd9cc6f331a3f0458fc7354b311ff3cd)